packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -4399,7 +4389,7 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
                     model,
                     column,
                     advancedFilterOptions,
-                    sortDirection,
+                    sortDirection as SortDirection,
```

Comment:
Guessing GWT can't express `Literal` types for sort either... rather than casting should we have a method to convert it and throw if it can't be converted? Would be more "correct".

Side note, not sure how this isn't an error currently, since `null` shouldn't match `SortDirection | undefined`

Reply:
Ya I think there were some issues with it/not a high priority. I'll add a typeguard and check for this.

`SortDirection` is `"ASC" | "DESC" | "REVERSE" | null``, so that's why it doesn't fail